File: app/src/main/java.backup/com/lumiyaviewer/lumiya/res/executors/HTTPFetchExecutor.java
package com.lumiyaviewer.lumiya.res.executors;

import com.lumiyaviewer.lumiya.GlobalOptions;
import java.util.concurrent.PriorityBlockingQueue;

public class HTTPFetchExecutor extends WeakExecutor {

    private static class InstanceHolder {
        /* access modifiers changed from: private */
        public static final HTTPFetchExecutor Instance = new HTTPFetchExecutor((HTTPFetchExecutor) null);

        private InstanceHolder() {
        }
    }

    private HTTPFetchExecutor() {
        super("ResourceHTTPFetch", GlobalOptions.getInstance().getMaxTextureDownloads(), new PriorityBlockingQueue());
    }

    /* synthetic */ HTTPFetchExecutor(HTTPFetchExecutor hTTPFetchExecutor) {
        this();
    }

    public static HTTPFetchExecutor getInstance() {
        return InstanceHolder.Instance;
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/res/executors/HTTPFetchExecutor.kt
package com.linkpoint.res.executors

import com.linkpoint.GlobalOptions
import java.util.concurrent.PriorityBlockingQueue

class HTTPFetchExecutor : WeakExecutor() {

    @JvmStatic
private class InstanceHolder {
        /* access modifiers changed from: private */
        const val HTTPFetchExecutor Instance = HTTPFetchExecutor((HTTPFetchExecutor) null)

        private InstanceHolder() {
        }
    }

    private HTTPFetchExecutor() {
        super("ResourceHTTPFetch", GlobalOptions.getInstance().getMaxTextureDownloads(), PriorityBlockingQueue())
    }

    /* synthetic */ HTTPFetchExecutor(HTTPFetchExecutor hTTPFetchExecutor) {
        this()
    }

    @JvmStatic
     fun getInstance(): HTTPFetchExecutor {
        return InstanceHolder.Instance
    }
}
